src/function/FunctionEditVars.ts

```diff
@@ -280,7 +280,11 @@ export function initElements() {
   return ids;
 }
 
-export function addToCount(vocabulary: string, ...terms: string[]) {
+export function changeVocabularyCount(
+  vocabulary: string,
+  changeFunction: (count: number) => number,
+  ...terms: string[]
+) {
   for (const term of terms) {
     WorkspaceVocabularies[vocabulary].count[Representation.FULL]++;
```

Comment:
@bindeali This line should probably apply the `changeFunction` as well, if I'm reading the code correctly. Otherwise LGTM.